### ISSUE TITLE
Single entry

### ISSUE THREAD

**Issue by ezekielchentnik** [cognitom/felt#11]
Is there a way to limit to a single entry point?

example:  /client/entry.js (rollup bundles from entry)
ignores everything else in /client

**Comment by cognitom** [comment]
Hi @ezekielchentnik. Yep, we can use `opts.patterns` option:

```javascript
// felt.config.js
module.exports = {
  patterns: ['client/entry.js']
}
```

Then, `$ felt --src client --recipe minimal --config felt.config.js`

See also https://github.com/cognitom/felt#optspatterns

**Comment by ezekielchentnik** [comment]
@cognitom does not seem to work.  when i specify the pattern, client/entry.js is not ran through rollup.

**Comment by ezekielchentnik** [comment]
@cognitom i've debugged and patterns solution works, however even when limiting by entry.js, files in sub folders are still copied,  Is there a way to completely ignore a whole folder?

**Comment by ezekielchentnik** [comment]
This appears to work:
```javascript
module.exports = {
  patterns: ['entry.js'],
  excludes: ['components/*.js']
}
```
```bash
/src
    /components
        App.jsx
    entry.js
```
```bash
felt --config felt.config.js --src src --export dist --debug
```

excludes seems to be un documented.  Also, I'm attempting to figure out a better pattern for all sub folders under src

**Comment by ezekielchentnik** [comment]
this pattern seems to work:
```javascript
module.exports = {
   excludes: ['!entry.js', '**/']
}
```

**Comment by cognitom** [comment]
@ezekielchentnik perfectly right! Sorry for my misleading.
Yeah, we should add `opts.exclude` to the docs.

> Also, I'm attempting to figure out a better pattern for all sub folders under src

I think, we could add `option.ignoreUnmatchedFiles` or something, but not self-explanatory...
Maybe, `--no-copy` option with `--export` could be simpler and readable. Thoughts?

**Comment by ezekielchentnik** [comment]
@cognitom an option to whitelist folders may be a better option?

i like the option of ```--no-copy``` for simple opting out.  

for more complex projects, maybe adding a whitelist  ```opts.include``` to make it easier for large folder structures where only a small number of folders are needed.

src:
```
/src
    /images
    /components
    /store
    /reducers
    /actions
    /middleware
    /constants
    /vendor
    /styles
    /utils
    entry.js
```
dist:
```
/dist
    /vendor
    /images
    global.css
    entry.js
```

felt.config.js
```javascript
module.exports = {
  patterns: ['entry.js', 'global.css'],
  exclude:[],
  include: ['images', 'vendor'],
  handlers: {
      '.js': rollup(),
      '.css': postcss()
    }
}
```

**Comment by cognitom** [comment]
Yeah, `opts.includes` seems popular option for this kind of tool. I like the idea. Thanks for your suggestion!
